src/components/workflow/_variables.scss

```diff
@@ -0,0 +1,31 @@
+@use '../../styles/variables' as colors;
+
+/**
+ * Themed colour variables for the flowchart component
+ */
+
+.kui-theme--light {
+  --node-details-fill: #{colors.$white-200};
+  --node-details-stroke: #{colors.$white-900};
+  --node-details-text: #{colors.$black-200};
+  --node-details-text-success: #{colors.$black-900};
+  --node-fill-success: #d8e8e0;
```

Comment:
these colours need a name, I've create a separate ticket for it https://github.com/kedro-org/kedro-viz/issues/2368